### PR TITLE
Avoid spinner when switching tabs in dashboard

### PR DESCRIPTION
**Pull request by philippotto**
I noticed that there was an incorrectly spinner every time one switches the active tab in the dashboard (which replaced the entire table).

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open dashboard and switch tabs --> table should **not** be replaced by a spinner for a brief moment

### Issues:
- follow-up to #6509

### CODE REVIEW

File: frontend/stylesheets/_utils.less [hunk 1]
```diff
@@ -122,3 +122,7 @@ td.nowrap * {
 .min-height-0 {
   min-height: 0;
 }
+
+.text-center {
```
Comment (hotzenklotz) [inline frontend/stylesheets/_utils.less:126]: This is not used in the PR. Do you still want to merge it?

Comment (philippotto) [reply frontend/stylesheets/_utils.less:126]: I noticed that the spinner in question was centered by using a `text-center` class which didn't exist (which didn't work obviously). That's why, I added this class.

File: frontend/javascripts/admin/admin_rest_api.ts [hunk 2]
```diff
@@ -2053,6 +2053,8 @@ export async function getPricingPlanStatus(): Promise<APIPricingPlanStatus> {
   return Request.receiveJSON("/api/pricing/status");
 }
 
+export const cachedGetPricingPlanStatus = _.memoize(getPricingPlanStatus);
```
Comment (hotzenklotz) [inline frontend/javascripts/admin/admin_rest_api.ts:2056]: Caching is not without its own problems. The status might update (e.g. new people will join, more datasets will be uploaded taking up space etc...) 
How does this cache live? Until the next page refresh, right? So, the above points might not be a such a big issue after all. 

The approach is fine by me.

**Review by hotzenklotz — APPROVED** [review]
LGTM.

File: frontend/stylesheets/_utils.less (hunk 1 above)
Comment (hotzenklotz) [reply frontend/stylesheets/_utils.less:126]: Ah. Good point. Maybe a Bootstrap leftover: https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment

File: frontend/javascripts/admin/admin_rest_api.ts (hunk 2 above)
Comment (philippotto) [reply frontend/javascripts/admin/admin_rest_api.ts:2056]: > Until the next page refresh, right?

Yes, exactly. When visiting the orga page, the non-cached version is used. So, only the dashboard is affected. If outdated data is noticed, the user will likely hit refresh to fix it.

**Pull request merged**